views/js/pciCreator/ims/mathEntryInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -176,13 +190,6 @@
         &ang; {{__ "angle"}}
     </label>
 </div>
-<div>
-    <label class="panel">
-        <input name="tool_integral" type="checkbox" {{#if tool_integral}}checked="checked"{{/if}}/>
-        <span class="icon-checkbox"></span>
-        {{__ "Indefinite integral"}}
-        </label>
-</div>
```

Comment:
**praise:** Nice catch! I was about to ask why this was removed, then I realized that it was a duplicate placed in the wrong place. 👍